app/controllers/api/storage.php

```diff
@@ -242,7 +242,7 @@
     ->param('fileId', '', new UID(), 'File unique ID')
     ->param('width', 0, new Range(0, 4000), 'Resize preview image width, Pass an integer between 0 to 4000.', true)
     ->param('height', 0, new Range(0, 4000), 'Resize preview image height, Pass an integer between 0 to 4000.', true)
-    ->param('gravity', Image::GRAVITY_CENTER, new WhiteList([Image::GRAVITY_CENTER, Image::GRAVITY_NORTH, Image::GRAVITY_NORTHWEST, Image::GRAVITY_NORTHEAST, Image::GRAVITY_WEST, Image::GRAVITY_EAST, Image::GRAVITY_SOUTHWEST, Image::GRAVITY_SOUTH, Image::GRAVITY_SOUTHEAST]), 'Image crop gravity', true)
+    ->param('gravity', Image::GRAVITY_CENTER, new WhiteList([Image::GRAVITY_CENTER, Image::GRAVITY_TOP, Image::GRAVITY_TOP_LEFT, Image::GRAVITY_TOP_RIGHT, Image::GRAVITY_LEFT, Image::GRAVITY_RIGHT, Image::GRAVITY_BOTTOM_LEFT, Image::GRAVITY_BOTTOM, Image::GRAVITY_BOTTOM_RIGHT]), 'Image crop gravity', true)
```

Comment:
Just realized we must mention the different options in the description as well. Maybe we can add a static method to get all the types in the Image library? `Image::getGravityTypes():array`

Reply:
We can.

Reply:
I've updated, but needs release of `utopia-php/image` to be updated as well

Reply:
Great, I have released v0.5.0